src/nvim/lua/vim.lua

```diff
@@ -261,14 +261,13 @@ end
 
 
 -- vim.fn.{func}(...)
-local function fn_index(t, key)
-  local function func(...)
+local function _fn_index(t, key)
+  t[key] = function(...)
     return vim.call(key, ...)
   end
-  t[key] = func
-  return func
+  return t[key]
 end
-local fn = setmetatable({}, {__index=fn_index})
+local fn = setmetatable({}, {__index=_fn_index})
```

Comment:
@bfredl JFI: renaming/rearranging these so that `gen_vimdoc.py` does not think they're "public" (and it makes semantic sense anyway)